python/tests/test_account.py

```diff
@@ -276,15 +276,23 @@ def test_get_set_profile_image_simple(self, ac1, data):
     def test_mute(self, ac1):
         chat = ac1.create_group_chat(name="title1")
         assert not chat.is_muted()
+        assert chat.get_mute_duration() == 0
         chat.mute()
         assert chat.is_muted()
+        assert chat.get_mute_duration() == -1
         chat.unmute()
         assert not chat.is_muted()
         chat.mute(50)
         assert chat.is_muted()
+        assert chat.get_mute_duration() <= 50
         with pytest.raises(ValueError):
             chat.mute(-51)
 
+        # Regression test, this caused Rust panic previously
+        chat.mute(2**63 - 1)
```

Comment:
should this not be `2**64 -1`?

Reply:
Duration is `i64`, because it has to be able to accept -1 for infinity.